### PR TITLE
Oved haruvi add properties to alert evidence

### PR DESCRIPTION
**Pull request by OvedHagaiHaruvi**
This is a new pull request based on this - https://github.com/microsoftgraph/microsoft-graph-docs/pull/21116/commits/1dc864c929d34bfb61734e8404678234019e75f5
Since too many changes at merge caused pc crash

Adding 3 new properties to alert evidences -

DetailedRoles - Detailed description of the entity role/s in an alert. Values are free-form.
DeviceEvidence.IpInterfaces - Ip interfaces of the device during the time of the alert.
RegistryValue.MdeDeviceId - A unique identifier assigned to a device by Microsoft Defender for Endpoint.

### CODE REVIEW

**Comment by OvedHagaiHaruvi** [comment]
@Danielabom  Please give a final review to this pr.
You've already approved the content in the pr mentioned above (which is the same )

File: api-reference/v1.0/resources/security-alertevidence.md [hunk 1]
```diff
@@ -54,10 +54,11 @@ This resource is the base type for the following evidence types:
...
 |remediationStatusDetails|String|Details about the remediation status.|
-|roles|[microsoft.graph.security.evidenceRole](#evidencerole-values) collection|One or more roles that an evidence entity represents in an alert. For example, an IP address that is associated with an attacker has the evidence role `Attacker`. Possible values are: `unknown`, `contextual`, `scanned`, `source`, `destination`, `created`, `added`, `compromised`, `edited`, `attacked`, `attacker`, `commandAndControl`, `loaded`, `suspicious`, `policyViolator`, `unknownFutureValue`. |
-|tags|String collection|Array of custom tags associated with an evidence instance. For example, to denote a group of devices or high value assets.|
+|roles|[microsoft.graph.security.evidenceRole](#evidencerole-values) collection|The role/s that an evidence entity represents in an alert, e.g., an IP address that is associated with an attacker will have the evidence role "Attacker".|
+|tags|String collection|Array of custom tags associated with an evidence instance, for example to denote a group of devices, high value assets, etc.|
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-alertevidence.md:61]:
```suggestion
|tags|String collection|Array of custom tags associated with an evidence instance, for example, to denote a group of devices, high-value assets, etc.|
```

File: api-reference/beta/resources/security-alertevidence.md [hunk 2]
```diff
@@ -56,10 +56,11 @@ This resource is the base type for the following evidence types:
...
 |remediationStatus|[microsoft.graph.security.evidenceRemediationStatus](#evidenceremediationstatus-values)|Status of the remediation action taken. The possible values are: `none`, `remediated`, `prevented`, `blocked`, `notFound`, `unknownFutureValue`.|
 |remediationStatusDetails|String|Details about the remediation status.|
-|roles|[microsoft.graph.security.evidenceRole](#evidencerole-values) collection|One or more roles that an evidence entity represents in an alert. For example, an IP address that is associated with an attacker has the evidence role `Attacker`. Possible values are: `unknown`, `contextual`, `scanned`, `source`, `destination`, `created`, `added`, `compromised`, `edited`, `attacked`, `attacker`, `commandAndControl`, `loaded`, `suspicious`, `policyViolator`, `unknownFutureValue`.|
-|tags|String collection|Array of custom tags associated with an evidence instance. For example, to denote a group of devices or high value assets.|
+|roles|[microsoft.graph.security.evidenceRole](#evidencerole-values) collection|The role/s that an evidence entity represents in an alert, e.g., an IP address that is associated with an attacker will have the evidence role `Attacker`.|
```
Comment (Danielabom) [inline api-reference/beta/resources/security-alertevidence.md:62]:
```suggestion
|roles|[microsoft.graph.security.evidenceRole](#evidencerole-values) collection|The role/s that an evidence entity represents in an alert, e.g., an IP address that is associated with an attacker will have the evidence role **Attacker**.|
```

File: api-reference/beta/resources/security-alertevidence.md [hunk 3]
```diff
@@ -56,10 +56,11 @@ This resource is the base type for the following evidence types:
...
+|tags|String collection|Array of custom tags associated with an evidence instance, for example to denote a group of devices, high value assets, etc.|
```
Comment (Danielabom) [inline api-reference/beta/resources/security-alertevidence.md:63]:
```suggestion
|tags|String collection|Array of custom tags associated with an evidence instance, for example, to denote a group of devices, high-value assets, etc.|
```

**Review by Danielabom — APPROVED** [review]
Thank you for working on this PR. PR approved.

**Pull request merged**